### PR TITLE
net-vpn/strongswan: version bump to 5.6.2

### PR DESCRIPTION
**Pull request by GrayTShirt**
This is also the upstream fix for:

Bug: https://bugs.gentoo.org/639654

### CODE REVIEW

File: net-vpn/strongswan/strongswan-5.6.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,303 @@
+# Copyright 1999-2017 Gentoo Foundation
```
Comment (mgorny) [inline net-vpn/strongswan/strongswan-5.6.1.ebuild]: Your calendar seems a bit off.

**Comment by GrayTShirt** [comment]
@mgorny should probably get merged sooner than later...

File: net-vpn/strongswan/strongswan-5.6.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,303 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+inherit eutils linux-info systemd user
+
+DESCRIPTION="IPsec-based VPN solution focused on security and ease of use, supporting IKEv1/IKEv2 and MOBIKE"
+HOMEPAGE="http://www.strongswan.org/"
```
Comment (candrews) [inline net-vpn/strongswan/strongswan-5.6.2.ebuild]: Please use https:// for the HOMEPAGE and SRC_URI

**Comment by mgorny** [comment]
I'm sorry but the current maintainers don't seem to care about their packages. If you're really interested in this, you could add yourself as proxied co-maintainer. This will permit us to merge updates from you.

**Comment by flappyports** [comment]
5.6.2 is in the Gentoo repo

**Pull request closed without merging**